src/application/index.tsx

```diff
@@ -1,6 +1,11 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
```

Comment:
Note that I'm leveraging `emotion`'s recommended pragma approach here, instead of chasing dragons through their babel config approach. The babel config option is super flakey when working with typescript and `ts-loader`, so I've dropped it. We can re-visit it in the future as needed, but for now adding `/** @jsx jsx */` and importing `jsx` are required in each file that uses `emotion`.

Reply:
What a bummer. But 🤷‍♀️, we can live with it.